models.py

```diff
@@ -63,3 +66,35 @@ class Meta:
         table = "autoresponses"
         table_description = "Represents the autoresponses for each GuildModel"
         unique_together = (("guild", "trigger"),)
+
+
+class ModerationRoles(CustomModel):
+    id = fields.IntField(pk=True)
+    admin_role = fields.BigIntField(description="ID of the Admin role", null=True)
+    mod_role = fields.BigIntField(description="ID of the Mod role", null=True)
+    moderation_role = fields.BigIntField(
+        description="ID of the General Moderation Role", null=True
+    )
+    guild_id = fields.BigIntField(description="Guild ID")
+
+    class Meta:
+        table = "staff_roles"
+        table_description = "Stores the roles for the moderation"
+
+
+class ModLogs(CustomModel):
+    id = fields.IntField(description="ID of the Case", pk=True)
+    guild_id = fields.BigIntField(description="ID of the Guild")
```

Comment:
Use guild as a foreign key rather than storing the guild_id.

Reply:
Haven't really understood about foreign keys a lot but will try

Reply:
Sure, good luck. You can take reference from my code in AutoResponse. Or DM me if you want further help. Btw, checkout tortoise orm docs too, they're good.